pkg/meta/meta.go

```diff
@@ -989,17 +989,27 @@ func (m *Meta) IterTables(dbID int64, fn func(info *model.TableInfo) error) erro
 	return errors.Trace(err)
 }
 
-// ListTables shows all tables in database.
-func (m *Meta) ListTables(dbID int64) ([]*model.TableInfo, error) {
+// GetDBMeta return all meta information of a database.
+// Note(dongmen): This method is used by TiCDC to reduce the time of changefeed initialization.
+// Ref: https://github.com/pingcap/tiflow/issues/11109
+func (m *Meta) GetDBMeta(dbID int64) ([]structure.HashPair, error) {
```

Comment:
```suggestion
func (m *Meta) GetMetasByDBID(dbID int64) ([]structure.HashPair, error) {
```
i don't have a better name, but i prev one doesn't desc what it does